app/src/test/shared/components/NavBarTests.tsx

```diff
@@ -17,18 +17,6 @@ describe("NavBar", () => {
         return shallow(<NavBarComponent crumbs={crumbs}/>).find("." + styles.chunk);
     };
 
-    it("can get props from store", () => {
-        navActions.navigate("a", "A");
```

Comment:
I think this test should stay, just instead of using `navigate` to set up the store, use `alt.bootstrap` (see e.g. `ChooseGroupContentTests` for an example of bootstrapping an alt js store for testing.)